server/routerlicious/packages/lambdas/src/alfred/index.ts

```diff
@@ -53,6 +53,19 @@ function getRoomId(room: IRoom) {
     return `${room.tenantId}/${room.documentId}`;
 }
 
+const getMessageMetadata = (document: string, tenant: string) => ({
+    documentId: document,
+    tenantId: tenant,
+});
+
+const handleServerError = async (logger: core.ILogger, errorMessage: string, document: string, tenant: string) => {
```

Comment:
Why are we naming the `document(Id)` and `tenant(Id)` params as just `document` and `tenant` in these 2 functions?

Reply:
Good point, I missed that. Renamed to `tenantId` and `documentId`.